fuse/utils/data/collate.py

```diff
@@ -126,25 +126,16 @@ def uncollate(batch: Dict) -> List[Dict]:
     else:
         batch_size = None
 
-        keys = batch.keys()
-
-        for key in keys:
-            if isinstance(batch[key], torch.Tensor):
+        for key in batch.keys():
+            if isinstance(batch[key], (torch.Tensor, np.ndarray, list)):
```

Comment:
@mosheraboh 

I didn't understand why we first check for Tensors and ndarrays separately so I changed it..